coursefinder/templates/coursefinder/partials/active_filters.html

```diff
@@ -1,18 +1,20 @@
 {% load static discover_uni_tags wagtailcore_tags %}
 
     <script>
-        // a function to select the value the passed in value for study_mode and set it to null
-        // if distance learning is the only filter applied, or if you are closing the oncampus filter, and the distance learning is active, all regional filters will be removed also.
         function closeFilter(filter) {  
             document.getElementById(filter).checked = false;
-            document.getElementById("sort_by_subject").value = false
+            if("{{search.total_courses}}" > "{{sort_by_subject_limit}}"){
```

Comment:
`{{search.total_courses}}` and `{{sort_by_subject_limit}}` look like Django template variables.

Do you need to have them here? I'm assuming it's done for speed? Could you at least make these a single load rather than loading them in to each function separately

Reply:
fair point, doing it now